### PR TITLE
nitty test changes 12/21

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: src/main/java/io/nats/client/ConsumerConfiguration.java [hunk 1]
```diff
@@ -577,6 +577,7 @@ public Builder rateLimit(int msgsPerSecond) {
         /**
          * Sets the maximum ack pending.
          * @param maxAckPending maximum pending acknowledgements.
+         * @return Builder
```
Comment (scottf) [inline src/main/java/io/nats/client/ConsumerConfiguration.java:580]: build was warning on javadoc

File: src/main/java/io/nats/client/NUID.java [hunk 2]
```diff
@@ -59,10 +59,10 @@
     static {
         try {
             srand = SecureRandom.getInstance("SHA1PRNG");
-            prand = new Random(bytesToLong(srand.generateSeed(8))); // seed with 8 bytes (64 bits)
         } catch (NoSuchAlgorithmException e) {
             throw new IllegalStateException("NUID requires SHA1PRNG Algorithm and it is not available.");
         }
+        prand = new Random(bytesToLong(srand.generateSeed(8))); // seed with 8 bytes (64 bits)
```
Comment (scottf) [inline src/main/java/io/nats/client/NUID.java:65]: moved outside of try/catch, nitty I know

File: src/test/java/io/nats/client/impl/TestMacros.java [hunk 3]
```diff
@@ -1,58 +0,0 @@
-package io.nats.client.impl;
```
Comment (scottf) [inline src/test/java/io/nats/client/impl/TestMacros.java:1]: moved package

File: src/test/java/io/nats/client/impl/AuthAndConnectTests.java [hunk 4]
```diff
@@ -13,60 +13,43 @@
...
-                assertTrue(Connection.Status.CLOSED == nc.getStatus(), "Connected Status");
-                nats.connect(false); // should do nothing
-                assertTrue(Connection.Status.CLOSED == nc.getStatus(), "Connected Status");
-                nats.reconnect(); // should do nothing
-                assertTrue(Connection.Status.CLOSED == nc.getStatus(), "Connected Status");
-            } finally {
-                nc.close();
-                assertTrue(Connection.Status.CLOSED == nc.getStatus(), "Closed Status");
-            }
+            NatsConnection nc = (NatsConnection)standardConnection(ts.getURI());;
```
Comment (ColinSullivan1) [inline src/test/java/io/nats/client/impl/AuthAndConnectTests.java:47]: extra `;`

**Review by ColinSullivan1 — APPROVED** [review]
LGTM.  Thanks!

**Pull request merged**